frontend/app/utils/env.server.ts

```diff
@@ -65,14 +70,8 @@ const serverEnv = z.object({
   JAVASCRIPT_ENABLED: z.string().transform(toBoolean).default('true'),
 
   // mocks settings
-  ENABLED_MOCKS: z
-    .string() // TODO :: GjB :: can this instead be a z.enum()
-    .transform((val) => val.split(',').map((val) => val.trim()))
-    .default(''),
-  MOCK_AUTH_ALLOWED_REDIRECTS: z
-    .string()
-    .transform((val) => val.split(',').map((val) => val.trim()))
-    .default('http://localhost:3000/auth/callback/raoidc'),
+  ENABLED_MOCKS: z.string().transform(emptyToUndefined).transform(csvToArray).refine(areValidMockNames).default(''),
```

Comment:
Why not `.default([])`?

Reply:
I read the Zod `.default` doccumentation.
https://zod.dev/?id=default

> Conceptually, this is how Zod processes default values:
> 
> 1. If the input is undefined, the default value is returned
> 2. Otherwise, the data is parsed using the base schema

Reply:
🤷